File: basic-calculator/js/bcalculator.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function main(){
    
    rl.question('Enter a number: ',(num1) =>{
        rl.question('Enter another number: ',(num2) => {
            console.log(`${num1} + ${num2} = ${parseInt(num1) + parseInt(num2)}`);
            console.log(`${num1} - ${num2} = ${parseInt(num1) - parseInt(num2)}`);
            console.log(`${num1} * ${num2} = ${parseInt(num1) * parseInt(num2)}`);
            console.log(`${num1} / ${num2} = ${parseInt(num1) / parseInt(num2)}`);
            rl.close();
        });
    });
}

main();

File: basic-calculator/java/bcalculator.java
import java.util.Scanner;

public class bcalculator {
    public static void main(String[] args) {
        int num1 = 0, num2 = 0;
        System.out.println("Enter a number: ");
        try (Scanner scanner = new Scanner(System.in)) {
            num1 = scanner.nextInt();
            System.out.println("Enter another number: ");
            num2 = scanner .nextInt();
        }
        System.out.println(num1 + " + " + num2 + " = " + (num1 + num2));
        System.out.println(num1 + " - " + num2 + " = " + (num1 - num2));
        System.out.println(num1 + " * " + num2 + " = " + (num1 * num2));
        System.out.println(num1 + " / " + num2 + " = " + ((double) num1 / num2));

    }
}